src/modules/env_var.rs

```diff
@@ -0,0 +1,49 @@
+use ansi_term::Color;
+use std::env;
+
+use super::{Context, Module};
+
+/// Creates a module with the value of the chosen environment variable
+///
+/// Will display the environment variable's value if all of the following criteria are met:
+///     - env_var.disabled is absent or false
+///     - env_var.variable is defined
+///     - a variable named as the value of env_var.variable is defined
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("env_var");
+    let module_style = module
+        .config_value_style("style")
+        .unwrap_or_else(|| Color::Black.bold().dimmed());
+
+    let env_name = module.config_value_str("variable");
+
+    // name is required
+    env_name?;
+
+    let default_value = module.config_value_str("default");
+
+    let env_value = match get_env_value(env_name.unwrap(), default_value) {
```

Comment:
This whole match is equivalent to: `get_env_value(env_name.unwrap(), default_value)?`

Reply:
Yea, this is way more elegant. I'm a total rookie in Rust, sorry.